### DISCUSSION TITLE
Doubt configuring auto responder in LiveHelperChat

### DISCUSSION THREAD

**Discussion by ts2050** [LiveHelperChat/livehelperchat#1957] [General]
We are setting up LiveHelperChat, but we are having difficulty with the 'auto responder' option. We would like it to send a message to the requester whenever they take more than 5 minutes to respond, and then close the chat after an additional 3 minutes. We understand that this can be achieved through the auto responder feature, both for the requester and for configuring the operator side. However, we have configured it several times and it has not been triggered under any circumstances. We're not sure what part we are missing or where we went wrong. Could someone help me?

I hope this helps! If you have any more questions or need further assistance, feel free to ask.

**Comment by remdex** [comment]
Auto reaponde stops always if customer leaves website. To auto mate chat closing you should use automatic chat closing
